coins/solana/Cargo.toml

```diff
@@ -2,27 +2,22 @@
 name = "walletd_solana"
 version = "0.1.0"
 edition = "2021"
-license = "MIT OR Apache-2.0"
-authors = ["The WalletD Team"]
-description = "Solana walletd library"
-homepage = "https://walletd.org/"
-repository = "https://github.com/walletd/walletd"
-readme = "README.md"
-categories = ["cryptography::cryptocurrencies"]
+
+# See more keys and their definitions at https://doc.rust-lang.org/cargo/reference/manifest.html
 
 [lib]
 name = "walletd_solana"
+crate-type = ["cdylib", "lib"]
 path = "src/lib.rs"
 
 [dependencies]
+async-trait = "0.1.60"
```

Comment:
async-trait is no longer required by the coin modules the async trait has been removed